### PR TITLE
Add fix lint script

### PR DESCRIPTION
**Pull request by bojanserafimov**
Runs black and isort from project root.

### CODE REVIEW

File: scripts/fix_lint.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright 2018-present Kensho Technologies, LLC.
+
+# Make sure the current working directory for this script is the root directory.
+cd "$(git -C "$(dirname "${0}")" rev-parse --show-toplevel )"
+
+# Print each command
+set -o xtrace
```
Comment (obi1kenobi) [inline scripts/fix_lint.sh]: `set -x` is a slightly more common way to do this.
```suggestion
set -x
```

File: scripts/fix_lint.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright 2018-present Kensho Technologies, LLC.
```
Comment (obi1kenobi) [inline scripts/fix_lint.sh]: This is very important to not get wrong, please be careful with copy-pasting.
```suggestion
# Copyright 2020-present Kensho Technologies, LLC.
```

Comment (chewselene) [inline scripts/fix_lint.sh]: not sure that this matters at all, but how do we pick what year goes here?

File: scripts/fix_lint.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (obi1kenobi) [inline scripts/fix_lint.sh]: These are just good practices for ~all shell scripts.
```suggestion

# Exit non-zero on errors, undefined variables, and errors in pipelines.
set -euo pipefail

# Ensure that the "**" glob operator is applied recursively.
# Make globs that do not match return null values.
shopt -s globstar nullglob

```

File: scripts/fix_lint.sh (hunk 2 above)
Comment (obi1kenobi) [reply scripts/fix_lint.sh]: The year the file was added, and yes — it matters a lot. Happy to chat if you're interested in the details.

Comment (bojanserafimov) [reply scripts/fix_lint.sh]: No idea. I usually use a year between the first commit date and now :D Will update this one to 2020

File: scripts/fix_lint.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+# Assert script is running inside pipenv shell
+if [[ "$VIRTUAL_ENV" == "" ]]
+then
+    echo "Please run pipenv shell first"
+    exit 1
+fi
```
Comment (obi1kenobi) [inline scripts/fix_lint.sh]: Consider reusing the approach we use elsewhere:
```
# Get into the graphql-compiler virtualenv. "pipenv shell" returns exit code 1 if already there,
# which doesn't matter to us so we avoid crashing the script by adding "|| true".
pipenv shell || true
```

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -114,7 +114,8 @@ If the style guides differ on a convention, the PEP 8 style guide is preferred.
 Additionally, any contributions must pass the linter :code:`scripts/lint.sh`
 when executed from a pipenv shell (i.e. after running :code:`pipenv shell`).
 To run the linter on changed files only, commit your changes and run
-:code:`scripts/lint.sh --diff`.
+:code:`scripts/lint.sh --diff`. Some linters can automatically fix errors.
+Use :code:`scripts/fix_lint.sh` to run the automatic fixes.
```
Comment (obi1kenobi) [inline CONTRIBUTING.rst]: @pmantica1 is there a second copy of this file that needs to be updated?

Comment (pmantica1) [reply CONTRIBUTING.rst]: `docs/source/about/contributing.rst`

File: scripts/fix_lint.sh (hunk 4 above)
Comment (bojanserafimov) [reply scripts/fix_lint.sh]: Doesn't work when outside pipenv shell, and prints confusing messages when already in pipenv shell

**Comment by bojanserafimov** [comment]
Maybe this should be a flag on `lint.sh` instead of a separate script. Thoughts?

**Comment by obi1kenobi** [comment]
> Maybe this should be a flag on `lint.sh` instead of a separate script. Thoughts?

Maybe, but there are kind of a lot of flags there already. Let's keep it simple.

**Pull request merged**